src/index.ts

```diff
@@ -514,7 +524,8 @@ export class BufferedChangeset implements IChangeset {
 
     let maybePromise = validationKeys.map(key => {
       const value: any = this[key];
-      const resolvedValue = value instanceof ObjectTreeNode ? value.unwrap() : value;
+      const resolvedValue =
+        value instanceof ObjectTreeNode ? value.unwrap() : this.maybeUnwrapProxy(value);
```

Comment:
Not sure if it is necessary here. Validation libraries like ember-validators can unwrap the proxy by themself.

Reply:
Oh I thought this was one place you requested.  I think we can still go with it?

Reply:
I think it should be mostly OK to leave it here.
The only gotcha is if the user had a validator relying on the fact that he gets a proxy.
So, it is potentially a breaking change.

Reply:
Gotcha gotcha.  I was referring to this test you put together.  I thought you were wanting the raw value passed through...

https://github.com/poteto/ember-changeset/pull/508/files#r440162709

Reply:
My original PR https://github.com/poteto/ember-changeset/pull/488 does the unwrapping by itself inside the validator. I took it from ember-validators https://github.com/offirgolan/ember-validators/blob/master/addon/presence.js